apps/condo/domains/acquiring/schema/Payment.js

```diff
@@ -381,6 +385,10 @@ const Payment = new GQLListSchema('Payment', {
                 }
                 const frozenFields = PAYMENT_FROZEN_FIELDS[oldStatus]
                 for (const field of frozenFields) {
+                    // We need to connect receipt after it's creation, if payment was made for virtual receipt
```

Comment:
Why?

Reply:
RegisterMultiPaymentForVirtualReceiptService. It's payments with qr-code, where bank data present, but no link for receipt in our system.

but we need that link to optimize allResidentBillingReceiptService